files/bootstrap.sh

```diff
@@ -142,6 +149,13 @@ Environment='KUBELET_EXTRA_ARGS=$KUBELET_EXTRA_ARGS'
 EOF
 fi
 
+if [[ "$ENABLE_DOCKER_BRIDGE" = "true" ]]; then
+    # Enabling the docker bridge network. We have to disable live-restore as it
+    # prevents docker from recreating the default bridge network on restart
+    jq '.bridge="docker0" | ."live-restore"=false' /etc/docker/daemon.json | tee  /etc/docker/daemon.json
```

Comment:
Passing `--enable-docker-bridge true` appears to override `/etc/docker/daemon.json` leaving it empty 🤔. I believe `tee` can cause [unexpected results](https://github.com/stedolan/jq/wiki/FAQ#general-questions) when used in combination with `jq`.

Reply:
I noticed that as well, this should be fixed in addition to the typo